spec/polymorphic_integer_type_spec.rb

```diff
@@ -26,6 +26,28 @@
       expect(link.target_type).to eq("Food")
     end
 
+    context "from HasManyReflection" do
+      it "sets the source properly from HasManyReflection" do
+        link_1 = Link.create()
+        link_2 = Link.create()
+        dog.source_links = [link_1, link_2]
+        expect(link_1.source_type).to eq("Animal")
+        expect(link_1.source_id).to eq(dog.id)
+        expect(link_2.source_type).to eq("Animal")
+        expect(link_1.source_id).to eq(dog.id)
+      end
+    end
+
+    context "from HasOneReflection" do
+      it "sets the source properly from HasManyReflection" do
```

Comment:
minor suggestion
```suggestion
      it "sets the source properly from HasOneReflection" do
```

typo here potentially? We can probably even leave out the `from ...` part since the text is included in the context